userguide/aws/how-tos/maintain-a-multi-tier-system.adoc

```diff
@@ -0,0 +1,177 @@
+= How to Maintain a Multi-Tier System
```

Comment:
We are going back and forth on this tradeoff. But I am not sure if this common file would fit cleanly in one of the four Divio categories.

Reply:
I'm not suggesting a separate file from the end user's perspective. So, your document would continue to look like it does now. 

AsciiDoc allows embedding a doc into another doc as follows:

```
== Introduction

include::{sourcedir}/includes/common.adoc[]
```

This would avoid duplication and make future changes easier (since changes would need to be done in one document only).

Hope this makes sense, please feel free to clarify if I've misunderstood!

Reply:
>  I think there are subtle differences in the "common" content that would make this difficult.

Okay, if moving this to a separate file is difficult, let's leave it as it is.  Wouldn't want to create more work for you. 

> Where would the includes be placed in the hierarchy so they do not appear to the end user?

Take a look at https://github.com/killbill/killbill-docs/tree/v3/userguide/subscription. The `userguide_subscription.adoc` simply points to the files within the `includes` directory.